components/script/dom/navigator.rs

```diff
@@ -38,6 +41,8 @@ pub struct Navigator {
     permissions: MutNullableDom<Permissions>,
     mediasession: MutNullableDom<MediaSession>,
     gpu: MutNullableDom<GPU>,
+    #[ignore_malloc_size_of = "Defined in wgpu"]
+    gpu_id_hub: RefCell<Option<Identities>>,
```

Comment:
I don't think we need an `Option` here. Each identity manager will be empty if not used, anyway.

Reply:
Thank you! I've updated the pull request with this change.